src/main/scala/com/databricks/spark/csv/readers/readers.scala

```diff
@@ -23,6 +23,21 @@ import java.io.StringReader
 import com.univocity.parsers.csv._
 
 /**
+  * Allows for greater extensibility
+  */
+trait BulkReader extends Iterator[Array[String]] {
+  protected def getReader(iter: Iterator[String]) = new StringIteratorReader(iter)
```

Comment:
It's not really a getter because I need to pass the `iter` arg. But perhaps I should rename the `def` to `reader`.